util/chunk/disk_test.go

```diff
@@ -64,30 +64,30 @@ func initChunks(numChk, numRow int) ([]*Chunk, []*types.FieldType) {
 	return chks, fields
 }
 
-func (s *testChunkSuite) TestListInDisk(c *check.C) {
+func TestListInDisk(t *testing.T) {
 	numChk, numRow := 2, 2
 	chks, fields := initChunks(numChk, numRow)
 	l := NewListInDisk(fields)
 	defer func() {
 		err := l.Close()
-		c.Check(err, check.IsNil)
-		c.Check(l.disk, check.NotNil)
+		assert.NoError(t, err)
+		assert.NotNil(t, l.disk)
 		_, err = os.Stat(l.disk.Name())
-		c.Check(os.IsNotExist(err), check.IsTrue)
+		assert.True(t, os.IsNotExist(err))
```

Comment:
require maybe? We should fail as quick as possible for such should-not-fail clean up.